support-frontend/app/views/subscriptionRedemptionForm.scala.html

```diff
@@ -29,6 +31,7 @@
       window.guardian = window.guardian || {};
       window.guardian.stage = @JsStringLiteral(stage);
       window.guardian.userCode = @JsStringLiteral(redemptionCode);
+      window.guardian.readerType = @JsStringLiteral.fromOption(maybeReaderType.map(_.toString));
```

Comment:
if we can add a suitable method instead of using toString, this would not be affected if we were to add an extra field or fail to compile if we change the ReaderType object in some incompatible way.  I'm especially nervous because this is the "zuora.api ReaderType" object but it's getting all over the place now

Reply:
I've realised there is already a `value` field on `ReaderType` so I will use that.